server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java

```diff
@@ -70,13 +72,14 @@
   private final OrganizationValidation organizationValidation;
   private final Configuration config;
   private final BuiltInQProfileRepository builtInQProfileRepository;
+  private final QualityGateFinder qualityGateFinder;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "qualityGateFinder" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)